lib/ansible/modules/web_infrastructure/wildfly.py

```diff
@@ -0,0 +1,309 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2016, Aditya C S <aditya.gnu@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'status': ['preview'], 'supported_by': 'community',
+                    'metadata_version': '1.0'}
+
+DOCUMENTATION = \
+    """
+module: wildfly
+version_added: "2.4"
+short_description: deploy applications to Wildfly
+description:
+  - Deploy applications to Wildfly domain and standalone mode
+options:
+  deployment:
+    required: true
+    description:
+      - The name of the deployment
+  src:
+    required: false
+    description:
+      - The remote path of the application ear or war to deploy
+  controller_address:
+    required: true
+    default: http://localhost:9990
+    description:
+      - The http api management address
```

Comment:
`http` should be `HTTP` and `api` should be `API`. The sentence should end with a period. The same in any other `description` in the `DOCUMENTATION` block.